docs/docs/cli_v2.md

```diff
@@ -1,19 +1,14 @@
-# CLI v2 Documentation
```

Comment:
weird, this is autogenerated differently. Try running `make precommit` again?

Reply:
Running `make precommit` indicates changes in the markdown file. I checked out the version from your main branch now.